app/controllers/customers_controller.rb

```diff
@@ -71,4 +80,33 @@ def set_customer
     def customer_params
       params.require(:customer).permit(:first_name, :last_name, :phone_number)
     end
+
+    def set_customer_detail(customer_params)
+      result = Customer.where(phone_number: customer_params[:phone_number]).any?
+      if result
```

Comment:
@krupa147 Lines 86 to 91 are confusing, it is not very clear that where we use DB record and where we are creating new one.